### DISCUSSION TITLE
自动应答客户端的 PTR 请求

### DISCUSSION THREAD

**Discussion by mengxinz** [IrineSistiana/mosdns#495] [Q&A]
reverse_lookup 支持 redis 缓存，更大的缓存时间，以及可以自动应答客户端的 PTR 请求。这个自动应答ptr请求具体怎么配置呢，求解

**Comment by QiuSimons** [comment]
https://irine-sistiana.gitbook.io/mosdns-wiki/mosdns/cha-jian-ji-qi-can-shu#reverselookup-ip-fan-cha-yu-ming-api-v4.2+-shi-yan-xing
